### DISCUSSION TITLE
[Question] Is is somehow possible to translate the command from pyarmor 7 to pyarmor 8+ (onefile and pack)

### DISCUSSION THREAD

**Discussion by JFuqX** [dashingsoft/pyarmor#1108] [Q&A]
So currently many people seem to have problems of actually using **pack** and **onefile** commands in Pyarmor 8+.

This is one of my old commands I used to compile back in Pyarmor 7:

`pyarmor-7 pack -x " --advanced 5" --clean -e "--onefile --icon 'ICON.ico' --add-data '/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/cloudscraper:cloudscraper' -c --hidden-import cloudscraper" --name TESTONEFILE /srcs/__main__.py`

So I was wondering if somebody has found a method to do something similar with Pyarmor 8+, Ive read trough the whole documentation but In the end I didn't felt smarter at all.

**Comment by jondy** [comment]
Refer to #1107
